qualtran/cirq_interop/_bloq_to_cirq.py

```diff
@@ -145,13 +146,15 @@ def _decompose_(self, qubits: Sequence[cirq.Qid]) -> cirq.OP_TREE:
         return self._decompose_with_context_(qubits)
 
     def _unitary_(self):
-        if self._num_qubits_() > 3:
-            # Prefer decomposition for large bloqs
-            return NotImplemented
```

Comment:
This had to be removed because it leads us to a situation where if we have a leaf bloq without a decomposition which acts on > 3 qubits but does override the `add_my_tensors` method; the cirq wrapper does not derive a unitary from it and raises an error which is not ideal

Reply:
sounds good